lib/util/number.js

```diff
@@ -257,3 +257,21 @@ exports.nearlyEqual = function(x, y, epsilon) {
   // Infinite and Number or negative Infinite and positive Infinite cases
   return false;
 };
+
+/**
+* Determines if n is a positive integer
+* @param {Number} n   Value to determine if it is a positive integer
+* @return {Boolean}   Whether the number is positive
+*/
+exports.isPositiveInteger = function(n) {
+  var BigNumber = require('../type/BigNumber');
+  if (this.isNumber(n) && this.isInteger(n) && n >= 0) {
+    return true;
+  }
+
+  if (n instanceof BigNumber && n.isInteger() && n.gte(0)) {
```

Comment:
I'm not sure whether it matters for handling `-0` (it's easy to add a test case for this though). 

But looking at the code of `isNegative` it is faster than using `gte`: it only checks the sign, whilst `gte` does a full comparison of all digits. So it may be nicer to use `isNegative` anyway.